README.md

```diff
@@ -5,9 +5,9 @@
 
 This plugin provides details about
 
-* actual bundle size
-* minified bundle size
-* gzipped bundle size
+* actual bundle size (approx runtime size)
```

Comment:
Not sure what `runtime` meant in that tweet. I'd call it "bundler parsing size" similar to the next point.